org.gnome.Calculator.json

```diff
@@ -1,25 +1,33 @@
 {
     "app-id": "org.gnome.Calculator",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.38",
+    "runtime-version": "40",
     "sdk": "org.gnome.Sdk",
     "command": "gnome-calculator",
     "finish-args": [
-        /* X11 + XShm access */
-        "--share=ipc", "--socket=fallback-x11",
-        /* Wayland access */
+        "--share=ipc",
+        "--socket=x11",
```

Comment:
Why x11 instead of fallback-x11?

Reply:
Whoops. Bad copy/paste from upstream. Didn't see this one.